bounties_api/std_bounties/client_helpers.py

```diff
@@ -47,7 +49,7 @@ def map_bounty_data(data_hash, bounty_id):
     except StatusError as e:
         if e.original.response.status_code == 504:
             redis_client.set('blacklist:' + str(bounty_id), True)
-            logger.error('Timeout for bounty id: ' + bounty_id)
+            logger.error('Timeout for bounty id: ' + str(bounty_id))
             raise e
```

Comment:
Noticed this from the last PR - we are now skipping `raise e` for any `status_code` that is not 504. I believe we should leave the `raise e` outside the `if` block so no exceptions are silenced.